assets/tests/components/notifications.test.tsx

```diff
@@ -257,12 +252,29 @@ describe("NotificationCard", () => {
     expect(wrapper.find(".m-notifications__card--new")).toHaveLength(0)
   })
 
-  test("clicking through opens VPP", () => {
+  test("clicking through opens VPP and hides notification", () => {
     const updatedNotification = {
       ...notification,
       tripIds: ["123", "456", "789"],
     }
-    mockUseReducerOnce(notificationsReducer, {
+    const dispatch = jest.fn()
+    const currentTime = new Date()
+    const openVPPForCurrentVehicle = jest.fn()
+
+    const wrapper = mount(
+      <NotificationCard
+        notification={updatedNotification}
+        dispatch={dispatch}
+        currentTime={currentTime}
+        openVPPForCurrentVehicle={openVPPForCurrentVehicle}
+      />
+    )
+    expect(openVPPForCurrentVehicle).not.toHaveBeenCalled()
+    expect(dispatch).not.toHaveBeenCalled()
+    wrapper.find(".m-notifications__card-info").simulate("click")
+    expect(openVPPForCurrentVehicle).toHaveBeenCalled()
+    expect(dispatch).toHaveBeenCalledWith({ type: "HIDE_LATEST_NOTIFICATION" })
+    /*    mockUseReducerOnce(notificationsReducer, {
```

Comment:
I think you meant to delete (rather than comment) this code?

Reply:
I sure did.